MiniSpace.Services.Reactions/src/MiniSpace.Services.Reactions.Application/Commands/Handlers/CreateReactionHandler.cs

```diff
@@ -9,25 +9,26 @@
 namespace MiniSpace.Services.Reactions.Application.Commands.Handlers
 {
     public class CreateReactionHandler(IReactionRepository reactionRepository,
-                                 IStudentRepository studentRepository,
                                  IPostRepository postRepository,
                                  IEventRepository eventRepository,
                                  IDateTimeProvider dateTimeProvider,
-                                 IMessageBroker messageBroker) : ICommandHandler<CreateReaction>
+                                 IAppContext appContext,
+                                 IMessageBroker messageBroker
+                                 ) : ICommandHandler<CreateReaction>
     {
         private readonly IReactionRepository _reactionRepository = reactionRepository;
-        private readonly IStudentRepository _studentRepository = studentRepository;
         private readonly IPostRepository _postRepository = postRepository;
         private readonly IEventRepository _eventRepository = eventRepository;
         private readonly IDateTimeProvider _dateTimeProvider = dateTimeProvider;
         private readonly IMessageBroker _messageBroker = messageBroker;
+        private readonly IAppContext _appContext = appContext;
 
         public async Task HandleAsync(CreateReaction command, CancellationToken cancellationToken = default)
         {
-            // Check if student exists
-            if (!await _studentRepository.ExistsAsync(command.StudentId))
-            {
```

Comment:
This check should not be deleted. It is necessary to ensure data synchronization across microservices.